convert-junit4-to-junit5/src/main/java/jb/Updater.java

```diff
@@ -1,68 +1,129 @@
 package jb;
 
-import java.io.*;
-import java.nio.file.*;
-import java.util.stream.*;
+import com.github.javaparser.JavaParser;
+import com.github.javaparser.ast.CompilationUnit;
+import jb.configuration.Configuration;
+import jb.configuration.JunitConversionLogicConfiguration;
+import jb.convert.ConversionOutcome;
+import jb.convert.ConversionResult;
+import jb.convert.ConversionResultBuilder;
+import jb.convert.JunitConversionLogic;
+import jb.convert.ast.CategoryClassToTagMetaAnnotationConversion;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.stream.Stream;
+
+import static java.util.stream.Collectors.toList;
 
 /**
  * Converts a file or nested directories to use JUnit 5 syntax where possible.
  * The resulting code may not compile, but it will be closer than the original
  * code so less to convert manually.
- * 
- * @author jeanne
  *
+ * @author jeanne
```

Comment:
I think we can remove this `@author` tag as it's a public repository (history will reveal) and there has changed a lot, so the current code was not written by her. I'm hope @boyarsky would not be angry at us.

Reply:
removed the author tag